webcompat/form.py

```diff
@@ -100,6 +114,15 @@ def get_problem(category):
     return u'Unknown'
 
 
+def get_browser_test(category):
```

Comment:
> What this function is supposed to do?
it seems to be the copy of get_problem but without any changes on it. Not finished?
What parameters does it accept, and what is supposed to return?

I did copy it from the existing code for the problem categories. It appeared to be used like an iterator to get the labels for the Yes/No radio buttons. But that whole section, with the formdata variable definition and build_formdata() function seemed a bit abstract. I tried looking up docs for WTForms but even then that section was not clear for me.

Do you mean that I should have updated the comment and argument names?